src/main/java/com/epam/ta/reportportal/dao/LaunchRepository.java

```diff
@@ -46,7 +46,8 @@
 	Stream<Long> streamIdsModifiedBefore(@Param("projectId") Long projectId, @Param("before") LocalDateTime before);
 
 	@Query(value = "SELECT l.id FROM Launch l WHERE l.status = :status AND l.projectId = :projectId AND l.lastModified < :before")
-	Stream<Long> streamIdsWithStatusModifiedBefore(@Param("projectId") Long projectId, @Param("status") StatusEnum status, @Param("before") LocalDateTime before);
+	Stream<Long> streamIdsWithStatusModifiedBefore(@Param("projectId") Long projectId, @Param("status") StatusEnum status,
```

Comment:
So why here we user Long and it works, but in test item repo BigInteger?

Reply:
It's a JPQL query, so it casts BigInteger to Long implicitly